site/content/docs/main/config/health-checks.md

```diff
@@ -44,6 +44,42 @@ Health check configuration parameters:
 - `timeoutSeconds`: The time to wait (seconds) for a health check response. If the timeout is reached the health check attempt will be considered a failure. Defaults to 2 seconds if not set.
 - `unhealthyThresholdCount`: The number of unhealthy health checks required before a host is marked unhealthy. Note that for http health checking if a host responds with 503 this threshold is ignored and the host is considered unhealthy immediately. Defaults to 3 if not defined.
 - `healthyThresholdCount`: The number of healthy health checks required before a host is marked healthy. Note that during startup, only a single successful health check is required to mark a host healthy.
+- `expectedStatuses`: An optional list of HTTP status ranges that are considered healthy. Ranges follow half-open semantics, meaning the start is inclusive and the end is exclusive. Statuses must be between 100 (inclusive) and 600 (exclusive).
+
+### Non-default expected statuses
+
+To consider status codes other than 200 healthy, specify `expectedStatuses` like the following:
```

Comment:
nit: one tiny thing to note would be that if you still want 200 to be valid you must make sure that is in your specified range(s)

Reply:
maybe also show an example of what the range should look like for allowing a specific status code (i.e what a range that consists of a solitary status code looks like)?

Reply:
Updated in https://github.com/projectcontour/contour/pull/5528/commits/b4f3752ce777da3f970c618708bea972c6978608